### PR TITLE
Adding hometown file for akorecky

### PR DESCRIPTION
**Pull request by akorecky**
@kousi2 - What kind of food you like so I can add them to my hometown file?

### CODE REVIEW

File: akorecky-chicago.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Chicago, IL
+### Resturants
+- Flight
+- Melting Pot
+
```
Comment (kousi2) [inline akorecky-chicago.md]: I like steak and Greek food. I also like Korean food.

**Review by kousi2 — CHANGES_REQUESTED** [review]
Add some restaurants that I would enjoy going to  and then we're good to go!

**Comment by akorecky** [comment]
@kousi2 - Added some restaurants, let me know if this good to :ship:

**Review by kousi2 — APPROVED** [review]
This is great, thanks!

**Pull request merged**